swap_meet/clothing.py

```diff
@@ -1,2 +1,14 @@
-class Clothing:
-    pass
\ No newline at end of file
+import uuid
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, id=None, condition=0, fabric="Unknown"):
+        super().__init__(id, condition)        
+        self.fabric = fabric
+
+    def get_category(self):
+        return "Clothing"
```

Comment:
You can use Python's special variables for this:

```suggestion
        return self.__class__.__name__  # Python sets this to "Clothing"
```

Reply:
In fact, since you do this in `Item`, you don't need to put the `get_category` overriding functions in your derived classes at all!